package.json

```diff
@@ -60,6 +63,9 @@
     "jsii-rosetta": "^1.80.0",
     "yargs": "^17.7.1"
   },
+  "engines": {
+    "node": ">= 14.18.0"
+  },
```

Comment:
Starting to wonder if we should not enforce engines at all by default. Especially for these smaller side projects, it seems unlikely people will be start complaining about `cdk-generate-synthetic-examples` not working with node10